src/Assetic/Filter/JSMinFilter.php

```diff
@@ -12,13 +12,14 @@
 namespace Assetic\Filter;
 
 use Assetic\Asset\AssetInterface;
+use JSMin\JSMin;
```

Comment:
instead of forcing to use the new version (which is a BC break), you should add support for both versions (by detecting whether the namespaced class exists)